lib/sawyer/agent.rb

```diff
@@ -31,21 +31,20 @@ def self.decode(data)
     # options  - Hash of options.
     #            :allow_undefined_methods  - Allow relations to call all the HTTP verbs,
     #                                        not just the ones defined.
-    #            :faraday                  - Optional Faraday::Connection to use.
+    #            :hurley                   - Optional Hurley::Client to use.
     #            :links_parser             - Optional parser to parse link relations
     #                                        Defaults: Sawyer::LinkParsers::Hal.new
     #            :serializer               - Optional serializer Class.  Defaults to
     #                                        self.serializer_class.
     #
-    # Yields the Faraday::Connection if a block is given.
+    # Yields the Hurley::Client if a block is given.
     def initialize(endpoint, options = nil)
       @endpoint = endpoint
-      @conn = (options && options[:faraday]) || Faraday.new
+      @client = (options && options[:hurley]) || Hurley::Client.new(@endpoint)
```

Comment:
I like `:client` :+1:  Hurley means nothing to me.  I'll replace him with Ben in a heartbeat!